src/mce/provider_mce.cpp

```diff
@@ -116,14 +129,21 @@ ScreenNs::ScreenNs()
     set_blanked_ = setter(prop);
 }
 
-void ScreenNs::set_blanked(bool v)
+void MceNs::reset_properties()
 {
-    set_blanked_(v ? "1" : "0");
+    setProperties(defaults_);
 }
 
-void MceNs::reset_properties()
+KeyboardNs::KeyboardNs()
+    : Namespace("maemo_InternalKeyboard")
```

Comment:
I guess we have to keep it named like that  (`maemo_` prefix) for legacy reasons?